pkgs/top-level/all-packages.nix

```diff
@@ -4936,6 +4936,8 @@ with pkgs;
 
   crossplane = with python3Packages; toPythonApplication crossplane;
 
+  crossplane-cli = callPackage ../applications/networking/cluster/crossplane-cli { };
```

Comment:
pleasemove the package to `pkgs/by-name` and remove this line

Reply:
Thanks for advice, fixed!